src/App.tsx

```diff
@@ -1,18 +1,64 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
+function getItems(
+  listOfItems: string[],
+  itemNumbers: string,
+): string[][] {
+  const result = [];
+  let itemsPart = [];
+  let count = 0;
+
+  for (let i = 0; i < listOfItems.length; i += 1) {
+    if (i === listOfItems.length - 1) {
+      itemsPart.push(listOfItems[i]);
+      result.push(itemsPart);
+      break;
+    }
+
+    itemsPart.push(listOfItems[i]);
+    count += 1;
+
+    if (count === +itemNumbers) {
+      result.push(itemsPart);
+      itemsPart = [];
+      count = 0;
+    }
+  }
+
+  return result;
+}
+
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState('5');
```

Comment:
The 'perPage' prop should be of type 'number'. This is because it is used in arithmetic operations, and it is better to avoid type coercion.
```suggestion
  const [perPage, setPerPage] = useState(5);
```